.vscode/cspell.json

```diff
@@ -1624,7 +1624,14 @@
         "Rpath",
         "rpath",
         "runpath",
-        "Cves"
+        "Cves",
+        "cves",
+	"Ocsp",
+	"ocsp",
+	"OCSP",
+	"sbom",
+	"Diffie",
+	"iotfirmwaredefense"
```

Comment:
as far as I know, this is case insensitive.
Maybe it should be fine if we only include `cves` `ocsp`, `sbom`, `diffie` and `iotfirmwaredefense`
And FYI per dotnet naming convention, the abbr longer than 3 (inclusive) should follow camel case - therefore if the generated API has `OCSP`, we should change them to `Ocsp`.